elide-core/src/main/java/com/yahoo/elide/core/sort/SortingImpl.java

```diff
@@ -25,25 +27,29 @@
  */
 @ToString
 @EqualsAndHashCode
-public class Sorting {
-
-    /**
-     * Denotes the intended sort type from json-api field.
-     */
-    public enum SortOrder { asc, desc }
+public class SortingImpl implements Sorting {
 
     private final Map<String, SortOrder> sortRules = new LinkedHashMap<>();
-    private static final Sorting DEFAULT_EMPTY_INSTANCE = null;
+    private static final SortingImpl DEFAULT_EMPTY_INSTANCE = null;
```

Comment:
Do we need this? Can simply return null in `parseSortRules`?

Reply:
This makes sorting consistent with pagination and allows us to introduce a default Sort annotation similar to what we have for Pagination in the future.